src/components/header/header.js

```diff
@@ -16,7 +17,9 @@ export default function(){
 				<li><Link activeClassName="active" to="/support">Support</Link></li>
 			</ul></nav>
 			{/*<span>Search bar here</span>*/}
-			<CarbonAd />
+			
 		</header>
+		<CarbonAd />
+		</>
```

Comment:
I'm fine with this change, do you mind argumenting it?

Reply:
At PR merged is fine too.

Reply:
Ahh, I moved the CarbonAd component out of Header component coz it messed up with the flex spacing for child-element nodes of the header 😇
Since CarbonAd component is always gonna be fixed, it can safely stay outside the header component.

Reply:
Adding a third component to the header, moves the nav to the left _(unless we explicitely tell it to stay on the right)_
![flex-space-even](https://user-images.githubusercontent.com/19433018/85234175-2d9acd00-b429-11ea-90aa-87fdfc6087e4.png)